waspc/data/Generator/templates/sdk/package.json

```diff
@@ -144,13 +142,29 @@
       "./webSocket/WebSocketProvider": "./dist/webSocket/WebSocketProvider.jsx",
 
       {=! ================= NEW API HERE =================== =}
-      {=! Public: { type MyEntity1, type MyEntity2, ... }  =}
-      "./server/api": "./dist/server/api/index.js"
+      {=! Public: { type MyEntity1, type MyEntity2, ... } =}
+      {=! Private: [] =}
+      "./server/api": "./dist/server/api/index.js",
+      {=! Public: { api }  =}
+      {=! Private: [sdk] =}
+      "./client/api": "./dist/api/index.js"
+   },
+   {=!
```

Comment:
Good to know 👍

Reply:
@sodic helped me with that, otherwise it would probably take me quite long to figure this out!